examples/pip_parse/WORKSPACE

```diff
@@ -0,0 +1,39 @@
+workspace(name = "example_repo")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "rules_python",
+    url = "https://github.com/bazelbuild/rules_python/releases/download/0.1.0/rules_python-0.1.0.tar.gz",
```

Comment:
I don't think this example works because `0.1.0` does not contain `pip_parse`

Reply:
Thats what I thought as well, but as @alexeagle pointed out the integration test runner replaces this string with a pointer to a local copy of the up to date repo.